.github/workflows/CI.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           - { name: "gio", features: "v2_66", nightly: "--all-features", test_sys: false }
           - { name: "glib", features: "v2_66", nightly: "--all-features", test_sys: false }
           - { name: "graphene", features: "", nightly: "", test_sys: true }
-          - { name: "pango", features: "v1_46", nightly: "--all-features", test_sys: false }
+          - { name: "pango", features: "v1_46", nightly: "--all-features", test_sys: true }
```

Comment:
:tada:

Reply:
heh i had to drop the commit; we need to figure out newer pango on CI first :/